settings.py

```diff
@@ -142,3 +142,15 @@
 
 with open(os.path.join(CERT_PRIVATE_DIR, CERT_DATA_FILE)) as f:
     CERT_DATA = yaml.load(f.read().decode("utf-8"))
+
+# Locale and Translations
+DEFAULT_LOCALE = 'en_US'
+DEFAULT_DATE_FORMAT = '%B %d, %Y'
+DEFAULT_TRANSLATIONS = {
+    'en_US': {
+        "success_text": u"has successfully completed a free online offering of",
+        "interstitial_intro": u"with",
+        "disclaimer": CERTS_SITE_DISCLAIMER_TEXT,
+        "verify_text": u"Authenticity can be verified at ",
```

Comment:
I wonder if we should include the link formatting in this string too instead of just concatenating it.

```
{
    'verify_text': u"Authenticity can be verified at {verification_link}",
}
```

This would allow for additional flexibility.